ci-operator/step-registry/abi/install/bmc/abi-install-bmc-commands.sh

```
@@ -0,0 +1,517 @@
+    typeset -i tryLeft="${OCP__ABI__WAIT__BOOTSTRAP__H}"
+    while ((tryLeft)); do
+        openshift-install agent wait-for bootstrap-complete && break
+        ((tryLeft--))
+    done
+    ((tryLeft))
+)
+cp -f "${OCP__ABI__CLUSTER_DIR}/auth/kubeconfig" "${SHARED_DIR}/kubeconfig-minimal"
+
+# Day-1.5 Phase.
+(
+    typeset cfgKey= cfgVal=
```

Shellcheck (0.11.0)

[warning] Remove space after = if trying to assign a value (for empty string, use var='' ... ).

(SC1007)